libs/designer/src/lib/ui/panel/templatePanel/quickViewPanel/tabs/summaryTab.tsx

```diff
@@ -11,10 +11,11 @@ import { ConnectorWithDetails } from '../../../../templates/connections/connecto
 import type { TemplatePanelTab } from '@microsoft/designer-ui';
 import { clearTemplateDetails } from '../../../../../core/state/templates/templateSlice';
 import Markdown from 'react-markdown';
+import { useDefaultWorkflowTemplate } from '../../../../../core/state/templates/templateselectors';
 
 export const SummaryPanel: React.FC = () => {
   const intl = useIntl();
-  const { manifest } = useSelector((state: RootState) => state.template);
+  const { manifest } = useDefaultWorkflowTemplate() ?? {};
```

Comment:
Should we supply a default inside the `useDefaultWorkflowTemplate` hook?  I see you writing `?? {}` a lot

Reply:
Because initially this UI is called when there are no templates selected. I need to make sure this does not initialize when template is not selected, then i can remove this.
And this code will be changed when we implement multi workflow, this is to make sure the current single workflow is not broken with new changes.